### DISCUSSION TITLE
Add a Power Amplifier (PA) to ESP32S3 RF Frontend

### DISCUSSION THREAD

**Discussion by lv333ming** [espressif/esp-iot-solution#438] [Ideas]
Can I extend ESP32's RF transmission power through an external Power Amplifier?

**Comment by Xiehanxin** [comment]
In theory, it's possible, but we don't have a ready-made demo. Currently, it's not highly recommended. you would need to handle tasks such as TX and RX switching, Wi-Fi and Bluetooth switching, and automatic gain control on your own.
